src/components/PlaylistCard/playlist-card.css

```diff
@@ -0,0 +1,45 @@
+.playlist-card-container{
+  width: 26rem;
+  height: 24rem;
+  background-color: var(--grey-background-color);
+  border: 1px solid var(--border-color);
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
+.playlist-count-container{
+  position: absolute;
+  background-color:var(--bg-color-secondary);
+  color: var(--secondary-color);
+  margin: 0;
+  padding: 0;
+  width: 60px;
+  height: 98%;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  right: 0;
+}
+
+.responsive-img{
+  height: 100%;
+}
+.playlist-description-box{
+  /* margin-left: 10px; */
+  display: flex;
+  align-items: center;
+  border-top: 1px solid var(--border-color);
+  width: 100%;
+}
+
+.playlist-description-box > p{
+  margin-left: 10px;
+}
+.empty-playlist-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 82%;
+}
+
+/* #21202070 */
```

Comment:
you can delete the unnecessary  commented code.

Reply:
removed.